src/lib/kit/components/Views/TimeRangeSelectorView/TimeRangeSelectorView.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+
+import cloneDeep from 'lodash/cloneDeep';
+
+import {ObjectIndependentView, ViewController, isStringSpec} from '../../../../core';
+
+const START_TIME = 'start';
+const END_TIME = 'end';
+
+export const TimeRangeSelectorView: ObjectIndependentView = ({
+    value,
+    spec,
+    name,
+    Layout,
+    ...restProps
+}) => {
+    const {startTimeSpec, endTimeSpec} = React.useMemo(() => {
+        let startTimeSpec, endTimeSpec;
+
+        if (spec.properties?.[START_TIME] && isStringSpec(spec.properties[START_TIME])) {
+            const _spec = cloneDeep(spec.properties[START_TIME]);
+
+            _spec.viewSpec.layout = 'row';
+
+            startTimeSpec = _spec;
+        }
+
+        if (spec.properties?.[END_TIME] && isStringSpec(spec.properties[END_TIME])) {
+            const _spec = cloneDeep(spec.properties[END_TIME]);
+
+            _spec.viewSpec.layout = 'row';
+
+            endTimeSpec = _spec;
```

Comment:
code is duplicated
let's do something like this

`
const [startTimeSpec, endTimeSpec] = [START_TIME, END_TIME].map((key) => {
   if (spec.properties?.[key] && isStringSpec(spec.properties[key])) {
            const spec = cloneDeep(spec.properties[key]);

            spec.viewSpec.layout = 'row';

            return spec;
        }
        
        return;
});
`